cypress/e2e/Class01.cy.js

```diff
@@ -0,0 +1,69 @@
+/// <reference types="Cypress" />
+
+describe('Validate EngageSphere website', ()=> {
+  beforeEach(()=> {
+    cy.visit('/')
+  })
+  context('Title', ()=> {
+    it('Validate title', ()=> {
+      cy.contains('h1','EngageSphere').should('be.visible')
+    })
+    it('Validate theme icon', ()=>{
+      cy.get('[id="theme-toggle-button"]').should('be.visible')
+    })   
+  })
+  context('Label', ()=>{
+    it('Validate default greting', ()=>{
+      cy.contains('Hi there!').should('be.visible')
+    })
+    it('Validate personal greting', ()=>{
+      cy.get('[data-testid="name"]').click().type('Caio')
+      cy.contains('Hi Caio').should('be.visible')
+    })
+  })
+  context('Modal customer details', ()=>{
+    it('Validate back button', ()=>{
```

Comment:
Que tal algo mais descritivo?

```suggestion
    it('goes back to the customer list when clicking the back button', ()=>{
```